components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -106,9 +106,7 @@ struct SwapParams {
   string buy_amount;
   string buy_token;
   string sell_token;
-  double buy_token_percentage_fee;
```

Comment:
I'm not sure if the JS client will error if it specifies these fields but doesn't use them anymore.  I think it's ok though if @onyb fixes in his next PR since the feature is disabled for now.